### DISCUSSION TITLE
⭐ Starstruck Achievements

### DISCUSSION THREAD

**Discussion by fcasamento** [Schweinepriester/github-profile-achievements#487] [Q&A]
Who can contribute to Starstruck I appreciate it, if you want to contribute to improving the terraform code, too. As I'm learning, there may be points of failure or better not having good practices 😅

https://github.com/fcasamento/primeiro-terraform

**Comment by bawejagb** [comment]


**Reply by fcasamento** [reply]
Of course friend @Aanvikshiki, contribute

**Comment by ArielMAJ** [comment]
I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many issues there.
